docs/architecture/baking.md

```diff
@@ -27,25 +27,62 @@ It packages pending operations into a block, signs it, and distributes it to the
 
```

Comment:
> The baking process includes many steps that Tezos users and developers don't need to think about, but at a basic level, baking a block follows these steps:
> 1. The protocol selects in advance a list of bakers for each block to create in a _cycle_, which is a certain number of blocks.
> 1. It also selects a list of bakers to act as validators.

- I understand that in Tenderbake/Tendermint the members of the consensus comitte are called `validators`. But this is confusing for a high-level doc.  We sometimes explain "baker" to non-Tezos folks as 'validators' because it is the generic term for block producers in PoS.

- Validators are **bakers**. Only bakers can be part of the consensus comitte,  
- Also, there is no separate selection here. Tenderbake selects a consensus commitee per level, and sorts the bakers by slot to determine the order.
 
> These bakers are responsible for attesting the blocks that other bakers create and publishing _attestations_ that the block is valid.
> 1. The first baker on the list has a certain amount of time to create the next block.
>  It packages pending operations into a block, signs it, and distributes it to the validators and other nodes.

It is distributed to everybody via the P2P network.

1. If the first baker doesn't publish a block in time, the next baker on the list has a certain amount of time to bake a block, and so on until a block is created.

> 1. The validators verify the block and publish their attestations in future blocks.
This sounds as if validators choose when to include them. Bakers send pre-attestations when they seen the block proposals, and attestation when they observe that a pre-quorum has been agreed (modulo rules on locking on proposals etc).  but they send them right away. This is the key into delays and higher rounds

> 1. The next list of bakers create the next block, until the end of the cycle.

This is missing the pre-attestation phase.  I'm not sure this is deliberate, or this is the cause of an original description based on Emmy that was patched to accomodate for Tenderbake.

Reply:
I added a note talking about pre-attestations and round, although I'm not sure these details are needed at this level. I kept "validators" as the "other" role of bakers, which translates in (pre-)attestations.